doc/geany.txt

```diff
@@ -3357,6 +3357,12 @@ Delete to line start            Ctrl-Shift-BackSpace      Deletes from the begin
                                                           current caret position.
 
 Duplicate line or selection     Ctrl-D                    Duplicates the current line or selection.
+							  If the selection spans more than one line,
+							  this duplicates the lines containing selections.
+							  If the selection spans only a single line, then
```

Comment:
As suggested to @b4n, "spans" is not the right word when its only part of a line.